common/djangoapps/third_party_auth/tasks.py

```diff
@@ -31,7 +31,6 @@ class MetadataParseError(Exception):
     pass
 
 
-@shared_task(name='third_party_auth.fetch_saml_metadata')
```

Comment:
Is removing `@shared_task` entirely the correct thing here?

Reply:
certainly not; that was a git mishap. good catch.